libs/wordpress-plugin/plugin/trunk/readme.txt

```diff
@@ -5,23 +5,24 @@ Tags: block protocol, blocks, gutenberg, gutenberg blocks, block, schema, countd
 Requires at least: 5.6.0
 Tested up to: 6.1.1
 Requires PHP: 7.4
-Stable tag: 0.0.3
+Stable tag: 0.0.4
 License: AGPL-3.0
 License URI: https://www.gnu.org/licenses/agpl-3.0.en.html
 
 Access a growing collection of high-quality and powerful blocks with a one-time plugin install.
 
 == Description ==
 
-The [Block Protocol](https://blockprotocol.org) plugin gives you an ever-growing set of high-quality blocks to use in your WordPress site. 
+Install the [Block Protocol](https://blockprotocol.org) plugin to access an ever-growing library of high-quality blocks within WordPress. 
 
-Discover and use new blocks when you need them, directly from within the WordPress Gutenberg Editor. No more installing new plugins to get more blocks.
+Discover and use new blocks when you need them, at the point of insertion, directly from within the WordPress editor. No more installing new plugins to get more blocks.
 
-In addition to useful blocks like Callouts, Headings, Images, and Quotes, the Block Protocol plugin provides powerful, new blocks for services such as OpenAI, and Mapbox.
+- Blocks that provide access to external services including **OpenAI** and **Mapbox**, without needing to sign up for these separately
+- Structured data blocks with SEO benefits for your website, helping you rank more highly in Google and Bing (e.g. the [Address](https://blockprotocol.org/@hash/blocks/address) and [How-to](https://blockprotocol.org/@hash/blocks/how-to) blocks)
+- Interactive blocks such as the [drawing](https://blockprotocol.org/@tldraw/blocks/drawing) and [countdown timer](https://blockprotocol.org/@hash/blocks/timer) blocks
+- "Traditional" content management blocks such as [callouts](https://blockprotocol.org/@hash/blocks/callout), headings, images, and quotes
```

Comment:
```suggestion
- "Traditional" content management blocks such as [callouts](https://blockprotocol.org/@hash/blocks/callout), [tables](https://blockprotocol.org/@hash/blocks/table), headings, images, and quotes
```

With the v2 (and now fixed) table block providing a (sometimes better) alternative to the WP native one, depending on what you're trying to do...